firefox_puppeteer/ui/menu.py

```diff
@@ -21,7 +21,7 @@ def menus(self):
         """
         menus = (self.marionette.find_element(By.ID, 'main-menubar')
                                 .find_elements(By.TAG_NAME, 'menu'))
```

Comment:
Given the new generic state of the class it would be great to have an additional unit test in the test_menu.py class which will cover a context menu like the #contentAreaContextMenu which exists for all pages in the content scope.